Module_4A/countries_data_handling.rb

```diff
@@ -0,0 +1,129 @@
+# ---------------- military module ---------------- 
+module MilitaryModule
+    attr_accessor :army_strength
+  
+    def initArmyStrength(army_strength)
+      @army_strength = army_strength
+    end
+  
+    def war_win_probability
+      return @army_strength > 300000
```

Comment:
no need to write return explicitly in ruby (Applicable to whole PR)

Reply:
Changed it to implicit code :+1: